### PR TITLE
[HBHE] Use uniform timeslew parameters

### PR DESCRIPTION
**Pull request by jaehyeok**
HCAL DPG wants to use uniform QIE timeslew parameters for ultra-rereco based on DN-17-036. The measurement was done with QIE8 in 2003, and the data was re-analyzed for a better extraction of parameters. Since there is no reason for QIE11 and QIE8 timeslews to be different, same parameters are used for both QIE8 and QIE11. 

The changes in cmssw are 

- The modifier for 2018_HE is removed 
- One set of parameters is used for all eras and for both HB and HE. 

This will affect both data and MC.

### CODE REVIEW

**Comment by mariadalfonso** [comment]
@tocheng, @lpernie this is one of the PR we need for the 10_4_0_preX release we want to close before the winter break/

**Comment by tocheng** [comment]
@cmsbuild please test

**Comment by tocheng** [comment]
@mariadalfonso This is only used in HLT, right?

**Comment by mariadalfonso** [comment]
@tocheng the updated parameters here, leave unchanged the HLT and change the offline (M2, MAHI)

**Comment by tocheng** [comment]
@mariadalfonso Thanks. From the code, it looks like the impact is on 2016 and 2017, right? Do you have any internal validation?

**Comment by hatakeyamak** [comment]
I think it will be nice to see the usual 50 GeV pion scan for 2016 and 2017 scenarios to assess the impact of this update.

**Comment by abdoulline** [comment]
Please, find below 50 GeV single-pion scan of Calorimetry w/wo this PR for 2016 and 2017 eras:

(1) 2016
https://cms-cpt-software.web.cern.ch/cms-cpt-software/General/Validation/SVSuite/HCAL/calo_scan_single_pi/10_X/TimeSlew/10_4_X_2016_updTimeSlew_vs_10_4_X_2016_SinglePi/

(2) 2017
https://cms-cpt-software.web.cern.ch/cms-cpt-software/General/Validation/SVSuite/HCAL/calo_scan_single_pi/10_X/TimeSlew/10_4_X_2017_updTimeSlew_vs_10_4_X_2017_SinglePi/

**Bottom line**:  changes are small (as expected) -
(i) somewhat better Chi2 for higher energy HB/HE RecHits;
(ii) energy scale change is < 1%.

**Comment by mariadalfonso** [comment]
@tocheng, @lpernie what is the status of this PR ?
we need it for the 10_4_0_pre5

Thanks you

**Comment by tocheng** [comment]
@mariadalfonso 
I will take a look at the performance plots and sign it today.
We will bring it to the CMSSW release meeting tomorrow @fabiocos

**Comment by tocheng** [comment]
+1

**Comment by fabiocos** [comment]
@mariadalfonso @jaehyeok @abdoulline the changes seem not to affect upgrade scenarios, even though the specific modifier is removed. Is this expected/intended?

**Comment by abdoulline** [comment]
Fabio, the change is back-propagated to <= 2017.

It was previously active for >= 2018 (for Run3, 2023 inheriting from 2018 
modifier).

**Comment by fabiocos** [comment]
@abdoulline thanks for the clarification, right, the modifier removed was imported in the global Run2 Era and from there up to upgrade.

**Comment by fabiocos** [comment]
+1
